pkg/db/db.go

```diff
@@ -154,15 +194,17 @@ func (d *db) getTableMeta(ctx context.Context) (meta tableMeta, _ error) {
 	return meta, err
 }
 
-func (d *db) doList(ctx context.Context, namespace, name *string, rev int64, after bool, cont, limit int64) (meta tableMeta, _ []record, _ error) {
+func (d *db) doList(ctx context.Context, namespace, name *string, rev int64, after bool, cont, limit int64, fields []string, vals []any) (meta tableMeta, _ []record, _ error) {
```

Comment:
Why isn't `vals` `[]string`? We only support strings, right?

Reply:
Using `[]any` allows the SQL to distiguish between `null` and empty values. I could use `*string` if you prefer.

Reply:
And it makes some of the code easier. I can do `append(s, vals...)` instead of casting each item in the slice.